clarifications to confidential clients and client authentication mechanism (#175)
oauth-wg/draft-ietf-oauth-attestation-based-client-auth: draft-ietf-oauth-attestation-based-client-auth.md

--- abstract

> This specification defines an extension to the OAuth 2.0 protocol {{RFC6749}} that enables a client instance to include a key-bound attestation when interacting with an Authorization Server or Resource Server. This mechanism allows a client instance to prove its authenticity verified by an client attester without revealing its target audience to that attester. It may also serve as a standalone OAuth 2.0 client authentication method.

Reviewer: ```suggestion
This specification defines an extension to the OAuth 2.0 protocol {{RFC6749}} that enables a client instance to include a key-bound attestation when interacting with an Authorization Server or Resource Server. This mechanism allows a client instance to prove its authenticity verified by a client attester without revealing its target audience to that attester. It may also serve as a mechanism for client authentication as per OAuth 2.0.
```
